src/client/l10n/locales.ts

```diff
@@ -39,6 +39,7 @@ export type LocaleData = {
     formatRegex: RegExp
     backendFormatExample: string
   }
+  currencyLocale: 'en-SE' | 'sv-SE' | 'en-NO' | 'nb-NO' | 'en-DK' | 'da-DK'
```

Comment:
isn't this really the standard/ISO locale? Maybe we should name it that.

And also use it as html-lang e.g.

Reply:
Yes but we already have another `IsoLocale`, which is the `Locale` type from the GraphQL schema, which uses `_` instead of `-`. So I made the decision to name this based on what we use it for. Maybe we should go over the naming of lots of these properties in a separate PR? 🤔

Reply:
But if we can use it for html-lang then it's awesome but I thought you only wanted to provide the language tag for that and not include country/region?

Reply:
aha, that might be the case indeed, okey -- let's keep it like this :)